test/integration/index.js

```diff
@@ -75,8 +78,26 @@ utils.assert(
   'App instances do not match.'
 );
 
+function updateRules() {
+  // Update database rules to the defaults. Rest of the test suite
+  // expects it.
+  const client = new apiRequest.SignedApiRequestHandler(defaultApp);
+  const dbUrl =  url.parse(defaultApp.options.databaseURL);
+  const defaultRules = {
+    rules : {
+      '.read': 'auth != null',
+      '.write': 'auth != null',
+    },
+  };
+  const headers = {
+    'Content-Type': 'application/json',
+  };
+  return client.sendRequest(dbUrl.host, 443, '/.settings/rules.json', 
+    'PUT', defaultRules, headers, 10000);
```

Comment:
Hmm, is it possible to prompt the developer with a yes or no question using something like https://nodejs.org/api/readline.html warning them of this issue before proceeding? A flag can be passed to bypass that if needed. What do you think?